rustfmt.toml

```diff
@@ -0,0 +1,26 @@
+# stable
+newline_style = "Unix"
+hard_tabs = false
+tab_spaces = 4
+# blank_lines_upper_bound = 0
+# blank_lines_lower_bound = 0
+# fn_args_layout = "Vertical"
+# force_multiline_blocks = true
+# imports_layout = "Vertical"
+# imports_granularity = "Item"
+# reorder_impl_items= true
+# reorder_modules = true
+# report_todo= "Always"
+# report_fixme = "Always"
+# spaces_around_ranges = true
+# trailing_comma = "Always"
+
+# unstable... should we require `rustup run nightly cargo fmt` ?
+# or just update the style guide when they are stable?
```

Comment:
I would not want to have nightly running in the CI because if it breaks it halts out development... But could be nice to have 
```
format_code_in_doc_comments = true
wrap_comments = true
```
in the config and so that we can run `rustup run nightly cargo fmt` locally before pushing but if it fails we do not enforce it. Is there a good way to set that up @apollo-pablo ?

Reply:
If we add them to the config it will fail when running without nightly, so maybe its not possible.